dataGenerator.js

```diff
@@ -41,12 +22,54 @@ let photoCounter = 1;
 
 let startTime = new Date;
 let endTime;
+let numberToWrite = 10000000;
+
+let reviews1 = '';
+let characteristics1 = '';
+let photos1 = '';
+
+let reviews2 = '';
+let characteristics2 = '';
+let photos2 = '';
+
+let reviews3 = '';
+let characteristics3 = '';
+let photos3 = '';
+
+let reviews4 = '';
+let characteristics4 = '';
+let photos4 = '';
 
-while (numberOfReviews <= 1000) {
+let reviews5 = '';
+let characteristics5 = '';
+let photos5 = '';
 
+let reviews6 = '';
+let characteristics6 = '';
+let photos6 = '';
+
+let reviews7 = '';
+let characteristics7 = '';
+let photos7 = '';
+
+let reviews8 = '';
+let characteristics8 = '';
+let photos8 = '';
+
+let reviews9 = '';
+let characteristics9 = '';
+let photos9 = '';
+
+let reviews10 = '';
+let characteristics10 = '';
+let photos10 = '';
+
+while (numberOfReviews < numberToWrite) {
+  // generate 4 random characteristics for each product
   let characteristicsToChoose = characteristicsPossiblities.slice();
   fourRandomCharacteristics = [];
   countdown = 3;
```

Comment:
should countdown have a more descriptive name?

Reply:
changed this to characteristicsCountdown